JetAnalyzers/bin/jet_response_fitter_x.cc

```diff
@@ -84,29 +98,32 @@ int main(int argc,char**argv)
   CommandLine cl;
   if (!cl.parse(argc,argv)) return 0;
   
-  string         input   = cl.getValue<string> ("input");
-  string         output  = cl.getValue<string> ("output",        "");
-  double         nsigma  = cl.getValue<double> ("nsigma",       1.5);
-  float          jtptmin = cl.getValue<float>  ("jtptmin",      1.0);
-  int            niter   = cl.getValue<int>    ("niter",          3);
-  int            ndfmin  = cl.getValue<int>    ("ndfmin",         5);
-  vector<string> algs    = cl.getVector<string>("algs",          "");
-  int            verbose = cl.getValue<int>    ("verbose",        0);
-  int            fittype = cl.getValue<int>    ("fittype",        0);
-  bool           doAbsRsp= cl.getValue<bool>   ("doAbsRsp",    true);
-  bool           doRelRsp= cl.getValue<bool>   ("doRelRsp",    true);
-  bool           doEtaRsp= cl.getValue<bool>   ("doEtaRsp",   false);
-  bool           doPhiRsp= cl.getValue<bool>   ("doPhiRsp",   false);
-  bool           doFlavor= cl.getValue<bool>   ("doFlavor",   false);
-  
+  string         input     = cl.getValue<string> ("input");
+  string         output    = cl.getValue<string> ("output",        "");
+  double         nsigma    = cl.getValue<double> ("nsigma",       1.5);
+  float          jtptmin   = cl.getValue<float>  ("jtptmin",      1.0);
+  int            niter     = cl.getValue<int>    ("niter",          3);
+  int            ndfmin    = cl.getValue<int>    ("ndfmin",         5);
+  vector<string> algs      = cl.getVector<string>("algs",          "");
+  int            verbose   = cl.getValue<int>    ("verbose",        0);
+  int            fittype   = cl.getValue<int>    ("fittype",        0);
+  bool           doAbsRsp  = cl.getValue<bool>   ("doAbsRsp",    true);
+  bool           doRelRsp  = cl.getValue<bool>   ("doRelRsp",    true);
+  bool           doEtaRsp  = cl.getValue<bool>   ("doEtaRsp",   false);
+  bool           doPhiRsp  = cl.getValue<bool>   ("doPhiRsp",   false);
+  bool           doFlavor  = cl.getValue<bool>   ("doFlavor",   false);
+  string         histName  = cl.getValue<string> ("histName");
```

Comment:
@Calpas Why is this histName option necessary? All of the AbsRsp, RelRsp, EtaRsp, or PhiRsp histograms in the file will be fit independent of their name (i.e. the Eta/RefPt bin).